packages/server/src/fhircast/routes.ts

```diff
@@ -71,65 +71,42 @@ publicSTU3Routes.get('/.well-known/fhircast-configuration', (_req: Request, res:
 });
 
 // Register a new subscription
+// Or publish an event depending on payload
 protectedCommonRoutes.post(
   '/',
-  [
-    body('hub.channel.type').notEmpty().withMessage('Missing hub.channel.type'),
-    body('hub.mode').notEmpty().withMessage('Missing hub.mode'),
-    body('hub.topic').notEmpty().withMessage('Missing hub.topic'),
-    body('hub.events').notEmpty().withMessage('Missing hub.events'),
-  ],
+  oneOf(
```

Comment:
This is a way to validation of two different types of payloads. https://express-validator.github.io/docs/api/one-of/

One of them is the "subscription request"
The other is the "context change request"